src/Button/ToggleButton/ToggleButton.spec.tsx

```diff
@@ -215,4 +215,14 @@ describe('<ToggleButton />', () => {
     wrapper.find('button').simulate('click');
     expect(wrapper.state('overallPressed')).toBe(true);
   });
+
+  it('can be rendered if iconName is set and no text or icon is set with the property pressed set to true', () => {
+    const wrapper = TestUtil.mountComponent(ToggleButton);
+    wrapper.setProps({
+      iconName: 'some-icon-name',
+      pressedIconName: undefined,
+      pressed: true
+    });
+    expect(wrapper).not.toBeUndefined();
```

Comment:
I'm not sure if this will test the actual result. You may want to check if the expected html fragment is actually being rendered through the wrapper.